stories/components/Error.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Error from '../../components/Error'
+import { MockedProvider } from '@apollo/react-testing'
+import GET_APP from '../../graphql/queries/getApp'
+
+export default {
+  components: Error,
+  title: 'Components/Error'
+}
+
+export const NotFound: React.FC = () => {
+  const mocks = [
```

Comment:
are all the mocks necessary? It seems that the `Error` component does not depend on any data received from a graphql request

Reply:
They were unnecessary indeed. Fixed.